woocommerce/woocommerce-android
# Product creation using package photo: Media picker library

This PR integrates the Media picker library for selecting an package photo. Supported media sources are: **Local device media**, **device camera** & **WP media library**. The image can be changed, but it is not scanned for keywords (next PR).

[Demo](https://github.com/woocommerce/woocommerce-android/assets/1522856/cf9f813e-58f8-4ad3-b831-a45351178ed9)

**To test:**
1. Go to Products
2. Tap on + FAB
3. Select the AI option
4. Tap on Use package photo button
5. Notice a media picker dialog with 3 options appears
6. Try each one and verify an image is properly displayed in the package photo bottom sheet
7. Once a photo is loaded, tap on the Pencil button above the image
8. Try changing the image and verify it works

*Note:* Please merge #10001 first.
